lldb/tools/yaml2macho-core/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+add_lldb_tool(yaml2macho-core
+    main.cpp
+    yaml2corespec.cpp
```

Nit: filename suggestions. 
```suggestion
    yaml2macho.cpp
    CoreSpec.cpp
```